plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt

```diff
@@ -173,9 +175,33 @@ internal class LocationPuckManager(
     vararg bearings: Double,
     options: (ValueAnimator.() -> Unit)? = null,
     forceUpdate: Boolean = false
+  ) {
+    if (settings2.puckBearingEnabled) {
+      animationManager.setEnabled(true)
+      animateToBearing(bearings, options, forceUpdate)
+    } else {
+      animateToBearing(
+        doubleArrayOf(0.0),
+        options = {
+          options?.invoke(this)
```

Comment:
actually not sure we need to use options provided from above, it should be sufficient to use zero duration + zero target bearing

Reply:
Yeah I guess we should discard any user-provided options at this point since we want a very specific behaviour. Good catch.